src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/merk/MerkPanel.tsx

```diff
@@ -18,7 +18,7 @@ import {
     harDelsvar
 } from '../../../utils/meldingerUtils';
 import { Meldingstype, Traad } from '../../../../../../../models/meldinger/meldinger';
-import { RadioPanelGruppe } from 'nav-frontend-skjema';
+import { RadioPanelGruppe, RadioProps } from 'nav-frontend-skjema';
```

Comment:
Tror det skal være `RadioPanelProps ` istedet for `RadioProps` her?

Reply:
Blir ikke `RadioPanelProps` mer rettet mot selve panelet og ikke mot Radio buttons?. `RadioPanelProps` ønsker en `onChange` osv..

Reply:
Panelet bruker `RadioPanelGruppeProps`, og er bare `label` som er påtvunget i `RadioPanelProps` tror jeg.
https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-skjema/src/radio-panel-gruppe.tsx#L9